### ISSUE TITLE
Didnt Download All The Images On The Post

### ISSUE THREAD

**Issue by Ntrs07** [Avnsx/fansly#194]
It didnt downloaded all the images, there are 72 images on the post and it only downloaded 5 images. Any Solutions?

**Comment by AlexMariacio** [comment]
It's not only not downloading all images, it also not downloading videos from posts...

**Comment by Animalm4st3r** [comment]
It stops exactly after the first 5 items in any post with more then 5 items in it, does not matter if its audio, video or pictures. It downloads the first 5 and then moves on to the next post. I know it did not behave like this before so i assume Fansly changed something with the last update.

**Comment by biggestsonicfan** [comment]
Encountering this issue as well

**Comment by luppyrou7** [comment]
Yup, Facing the same issue, pinned videos are also not getting downloaded, please look into it Thanks!

**Comment by e2489** [comment]
Same issue. Hoping for a fix!

**Comment by biggestsonicfan** [comment]
Looks like this is a duplicate of #193, so I will continue discussion there.
